shared/common-adapters/icon.js.flow

```diff
@@ -22,6 +22,11 @@ export type Props = {
   inheritColor?: boolean,
   underlayColor?: string,
   className?: string,
+  // External services use custom colors which aren't defined in the stylesheet
+  // So they break the css optimizations. We have this extra prop so we can set
+  // the color without messing with the existing color code.
+  // TODO would be nice to not have to do this.
+  externalServiceColor?: Color,
```

Comment:
confused why we need this? why aren't we just setting color?

Reply:
likely easier to chat about this but. 
I think this is all temporary. we aren't going to supply these as colors to iconfonts soon. soon it'll all be images. therefore this is all going away. 
short term i'm fine with this approach but. 1 we should comment that its temporary and 2. i'd prefer to make this more generic like `this.props.inlineColorOverride` or something. not something search specific language inside icon